### DISCUSSION TITLE
Can I use it in my Android application written in Java?

### DISCUSSION THREAD

**Discussion by theoyuncu8** [patrykandpatrick/vico#619] [Questions]
Can I use it in my Android application written in Java?

**Comment by patrickmichalik** [comment]
Hello! Please see #299.

**Comment by theoyuncu8** [comment]
> Hello! Please see #299.
Thank you for your answer. So, do you have any thoughts of doing it only with Kotlin? At least there are no problems when using libraries built with Kotlin. But I think I may have problems with compose.

Do you have an example with java?

**Reply by patrickmichalik** [reply]
Because Vico is written in Kotlin, and Kotlin is the language Google recommends for Android development, we don’t have any Java examples, I’m afraid. I’d definitely suggest using Kotlin. Indeed, it’ll be easier to use Vico and many other libraries. Regarding Compose, there’s no need to use it—you can use Vico with Kotlin and the view system.
